<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PCCullClass</class>
 <widget class="QMainWindow" name="PCCullClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PCCull</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="PCDirEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="PCPatternEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="DemDirEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="DemPatternEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ElevThreshEdit">
    <property name="geometry">
     <rect>
      <x>372</x>
      <y>300</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="PCDirButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DemDirButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CancelButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>360</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>360</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Directory</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pattern (eg *.las)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Directory</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pattern (eg *.tif)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Input all point cloud files from directory that match pattern</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>501</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Input all DSM files that match pattern to cull against -- only last-hit (a2) DSMs should be culled against</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Elevation threshold for culling near DSM (m)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Output point cloud files will have modified name (eg input test.las -&gt; output test_cull.las)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pcFoundEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="demFoundEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>441</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Found</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Found</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>521</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>521</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>555</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PCCull.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>PCDirButton</sender>
   <signal>clicked()</signal>
   <receiver>PCCullClass</receiver>
   <slot>doPCDirBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>493</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DemDirButton</sender>
   <signal>clicked()</signal>
   <receiver>PCCullClass</receiver>
   <slot>doDemDirBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CancelButton</sender>
   <signal>clicked()</signal>
   <receiver>PCCullClass</receiver>
   <slot>doCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>519</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>PCCullClass</receiver>
   <slot>doApply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PCPatternEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>PCCullClass</receiver>
   <slot>doPcPattern()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>461</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DemPatternEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>PCCullClass</receiver>
   <slot>doDemPattern()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>518</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>doPCDirBrowse()</slot>
  <slot>doDemDirBrowse()</slot>
  <slot>doCancel()</slot>
  <slot>doApply()</slot>
  <slot>doPcPattern()</slot>
  <slot>doDemPattern()</slot>
 </slots>
</ui>
